scripts/lint.sh

```
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+./gradlew \
```

Should we just use the one that's in the pipelines directory?
https://github.com/govuk-one-login/mobile-android-pipelines/blob/6891a2f654ed633ffb28abf2e665ae52a305a3c3/scripts/lintAndTestProject.sh

---

The one in the pipelines directory fails as soon as any task fails because it doesn't have the `--continue` flag - this might be how others prefer it, but I find myself wanting to use the `--continue` flag so that any and all tasks that would fail are shown

---

That makes sense, and I think you're right - on the pipelines we'd want this to happen to save build minutes and fail fast. I wonder if there's a way to pass in the continue flag into the script so we don't need to duplicate it?

---

Am I OK to merge this PR for convenience and then update our approach once we decide on how best to use the shared scripts?